apps/easypid/README.md

```diff
@@ -0,0 +1,133 @@
+TODO if possible: add pictures / video
+ <div align="center">
+  <img src="assets/ios-1.jpg" width="30%" />
+  <img src="assets/ios-2.jpg" width="30%" /> 
+  <img src="assets/ios-3.jpg" width="30%" />
+</div> -->
+<!-- 
+<p align="center"><i>Impression of Paradym Wallet</i></p> -->
+
+## Install
+
+
+## Try it out
+
+Here's some resources and tips that might be helpful while testing the app.
+
+### Before you start
+- Make sure you are connected to the VPN <!--TODO: LINK -->
+- Have an eID card ready
+- Have the [test relying party](https://funke.animo.id/) ready
+    <!-- - The test relying party enables you to select a TODO ADD -->
+
+### During
+- The very first screen has an option to switch between the C and B' flow for testing purposes. It is located on the left side besides the continue button.
+- There is an option to reset the wallet during testing. It is located in the menu that you can find on the home page. 
+
+
+## Project Structure
+
+<!-- The project is a monorepo managed using **pnpm**, which contains an **Expo React Native** application. The UI is built using **Tamagui**, and navigation is handled using **Expo Router, React Navigation and Solito**. For the Agent and SSI capabilities **Aries Framework JavaScript (AFJ)** is used.
+
+The folder structure is as follows
+
+- `apps` top level applications
+  - `paradym` Paradym Wallet - react native app for iOS & Android
+  - `easypid` EasyPID Wallet - react native app for iOS & Android
+- `packages` shared packages
+  - `ui` includes our custom UI kit that will be optimized by Tamagui
+  - `agent` includes the Aries Framework JavaScript (AFJ) agent and SSI capabilities
+  - `app` you'll be importing most files from `app/`
+    - `features` (don't use a `screens` folder. organize by feature.)
+    - `provider` (all the providers that wrap the app, and some no-ops for Web.)
+    - `navigation` This folder contains navigation-related code for RN. You may use it for any navigation code, such as custom links.
+
+You can add other folders inside of `packages/` if you know what you're doing and have a good reason to. -->
+
+## Tech stack / base components
+
+The following components were used.
+- [Credo](https://github.com/openwallet-foundation/credo-ts)
+
+The following standards and specifications were implemented.
+- [OpenID for Verifiable Credential Issuance - ID 1 / Draft 13](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-ID1.html)
+- [OpenID for Verifiable Presentations - Draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0-20.html)
+- [SD-JWT VC - Draft 3](https://www.ietf.org/archive/id/draft-ietf-oauth-sd-jwt-vc-03.html)
+- [Self-Issued OpenID Provider V2 - Draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0-13.html)
+- [ISO 18013-5](https://www.iso.org/standard/69084.html)
+- [ISO/IEC TS 23220-3 - WD13](https://www.iso.org/standard/86783.html)
+- [ISO/IEC TS 18013-7 DTS Ballot Text](https://www.iso.org/standard/82772.html)
```

Comment:
@auer-martin i think we support this except for JARM right?

Reply:
yes jarm and the issuersigned